### PR TITLE
[CDAP-17693] Introduce spark 3 for tests, drop spark 1, create spark …

### PR DESCRIPTION
**Pull request by tivv**
…3 runtime for cdap sandbox and standalone

### CODE REVIEW

File: format-avro/pom.xml [hunk 1]
```diff
@@ -112,8 +112,8 @@
         <artifactId>cdap-maven-plugin</artifactId>
         <configuration combine.self="override">
           <cdapArtifacts>
-            <parent>system:cdap-data-pipeline[6.1.0-SNAPSHOT,7.0.0-SNAPSHOT)</parent>
-            <parent>system:cdap-data-streams[6.1.0-SNAPSHOT,7.0.0-SNAPSHOT)</parent>
+            <parent>system:cdap-data-pipeline2_2.11[6.1.0-SNAPSHOT,7.0.0-SNAPSHOT)</parent>
```
Comment (albertshau) [inline format-avro/pom.xml]: these don't need to change. The jars get renamed to cdap-data-pipeline in the corresponding spark folder in the artifacts directory, so they are always in the system as 'cdap-data-pipeline'

Comment (tivv) [reply format-avro/pom.xml]: Got you, reverted

**Review by albertshau — APPROVED** [review]

**Pull request merged**